src/pages/InviteReportParticipantsPage.tsx

```diff
@@ -187,10 +186,25 @@ function InviteReportParticipantsPage({betas, personalDetails, report, didScreen
         return OptionsListUtils.getHeaderMessage(invitePersonalDetails.length !== 0, Boolean(userToInvite), searchValue);
     }, [searchTerm, userToInvite, excludedUsers, invitePersonalDetails, translate, reportName]);
 
+    const footerContent = useMemo(
+        () => (
+            <FormAlertWithSubmitButton
+                isDisabled={!selectedOptions.length}
+                buttonText={translate('common.invite')}
+                onSubmit={inviteUsers}
+                containerStyles={[styles.flexReset, styles.flexGrow0, styles.flexShrink0, styles.flexBasisAuto]}
+                enabledWhenOffline
+                disablePressOnEnter
+            />
+        ),
+        [selectedOptions.length, inviteUsers, translate, styles],
+    );
+
     return (
         <ScreenWrapper
             shouldEnableMaxHeight
             testID={InviteReportParticipantsPage.displayName}
+            includeSafeAreaPaddingBottom={false}
```

Comment:
Just curious, what does this do and why do we want it to be false here?

Reply:
Can you explain it to me like I am a child? 😄 Just to make sure I understand what it does and why we are setting it to false. Thanks!

Reply:
Got it, but don't we actually want that SafeArea on iOS below the button even when there is no offline indicator? Otherwise the button might be too close to the home bar. For instance, on desktop you see 20px below the button: 
![CleanShot 2024-06-03 at 16 56 05@2x](https://github.com/Expensify/App/assets/2319350/e96cfa89-ef69-4f51-8ea0-d954e31749b2)

But on iOS, that 20px is being occupied by the homebar:
![CleanShot 2024-06-03 at 16 56 37@2x](https://github.com/Expensify/App/assets/2319350/e383de3d-20fc-478c-bccb-f25cec2c96f0)

I would think there would be SafeArea and then the 20px on top of it.

Reply:
Yeah I'm kind of surprised we would not be including SafeArea at the bottom of iOS views. Can you show me what this looks like when we set it to true?

Reply:
Yeah, I think that feels more expected to be TBH. Curious if @Expensify/design agrees though. Maybe you can share a side-by-side set of screenshots for easier comparison.

Reply:
Cool, yeah I think that helps validate that we should always be using true for iOS/Android.

Reply:
I'm fine if we have a new issue for this one so we don't block progress on this PR.